cli/src/action/circuit/mod.rs

```diff
@@ -59,16 +62,41 @@ impl Action for CircuitCreateAction {
             }
         }
 
-        let services = match args.values_of("service") {
-            Some(services) => services,
-            None => return Err(CliError::ActionError("Service is required".into())),
-        };
+        #[cfg(feature = "circuit-template")]
+        #[allow(clippy::single_match)]
+        match args.value_of("template") {
```

Comment:
Why not `if let Some(template_name) = args.value_of("template") { ... }`?

Reply:
I tried that but was getting this error. I could not find another way around it, other than using `match`.

```
error: attributes are not yet allowed on `if` expressions
   --> cli/src/action/circuit/mod.rs:67:9
    |
67 |                #[cfg(feature = "circuit-template")]
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error
```

Reply:
It the `if let` statement is not returning a value, you could wrap it in a block:
```rust
#[cfg(feature = "circuit-template")]
{
    if let Some(template_name) = args.value_of("template") {
        ...
```

Reply:
Addressed